File: OCH_Tao/ABC/B/091.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const N = parseInt(input.shift(), 10);
const S = [];
for (let i = 0; i < N; i++) {
  S.push(input.shift());
}
const M = parseInt(input.shift(), 10);
const T = [];
for (let i = 0; i < M; i++) {
  T.push(input.shift());
}
const union = (a, b) => {
  let A = new Set(a);
  let B = new Set(b);
  for (const i of B) {
    A.add(i);
  }
  return A;
}
const L = [...union(S, T)];
let x = 0;
for (const i of L) {
  let cnt = 0;
  for (const A of S) {
    if (i === A) {
      cnt++;
    }
  }
  for (const B of T) {
    if (i === B) {
      cnt--;
    }
  }
  x = Math.max(x, cnt);
}
console.log(x);

File: OCH_Tao/ABC/B/091.py
N = int(input())
S = []
for i in range(N):
  S.append(input())
M = int(input())
T = []
for i in range(M):
  T.append(input())
L = list(set(S)|set(T))
X = 0
for i in range(len(L)):
  cnt = 0
  cnt += S.count(L[i])
  cnt -= T.count(L[i])
  X = max(cnt,X)
print(X)